### PR TITLE
fix comments for PR #615

### PR DESCRIPTION
**Pull request by SabraO**

### CODE REVIEW

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java [hunk 1]
```diff
@@ -2213,4 +2213,114 @@ public void addApplicationContext(Connection dbConnection, int tenantId, int ver
...
+        try {
+            preparedStatement = dbConnection.prepareStatement(SQLQueryConstants.GET_MATCHING_VERSION_COUNT);
+            preparedStatement.setInt(1, tenantId);
+            preparedStatement.setString(2, versionName);
+            preparedStatement.setString(3, applicationName);
+            preparedStatement.setInt(4, tenantId);
+            resultSet = preparedStatement.executeQuery();
+
+            if (resultSet.next()) {
+                return (resultSet.getInt(SQLQueryConstants.MATCHING_VERSION_COUNT) > 0);
```
Comment (danushkaf) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Why not move the logic to manager and just return the count from dao. DAO is for crud operations right? Manager will handle the logic. WDYT?

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java [hunk 2]
```diff
@@ -213,6 +215,21 @@
 
     public static final String GET_ALL_CLOUDS = "SELECT * FROM AC_CLOUD";
 
+    //"COLLATE utf8_bin" is added to perform a case sensitive search on the name
+    public static final String GET_MATCHING_ENV_VARIABLE_COUNT = "SELECT COUNT(id) as matching_env_variable_count " +
```
Comment (mcvidanagama) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java]: cant we use MATCHING_ENV_VARIABLE_COUNT in all the places ?

Comment (SabraO) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java]: @mcvidanagama: The question is a bit unclear. Could you pls clarify the question?

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java (hunk 1 above)
Comment (SabraO) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Fixed @ dfd2c14e26d8655a97a9f44a95b72504381b1898

**Pull request merged**